src/melee/ft/chara/ftNess/ftNess_AttackHi4.c

```diff
@@ -353,22 +342,18 @@ void ftNess_YoyoApplySmash(
     fp->x2114_SmashAttr.x212D = (u8) ((struct Unknown*) lbl_804D6528)->x4;
     fp->x2114_SmashAttr.x2128 = colAnimID;
     smashColAnimID = smash_attr->x2128;
-    if (smash_attr->x2128 != 0x7B) {
+
+    if (smash_attr->x2128 != 0x7B)
         func_800BFFD0(fp, smashColAnimID, 0);
-    }
+
     func_8007EBAC(fp, 0x24U, 0U);
     func_80088510(fp, 0x334A7, 0x7F, 0x40);
 }
 
-extern void func_802C0010(HSD_GObj*, void*); /* extern */
-
-const Vec3 YoyoChargePos = { 0.0f, 0.0f, 0.0f };
+static Vec3 const YoyoChargePos = { 0 };
```

Comment:
I think this looks better for zeroing structs but let me know what you think.

Reply:
Yeah I believe ` = { 0 }` is a common idiom in C to zero a struct. Looks good